.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+# enable next line when tests are fixed
+# npm test
```

Comment:
Currently there are broken tests when I run `npm test`
Disabled `npm test` for now. Can be re-enabled when they are fixed.